app/components/modals/SearchModal.tsx

```diff
@@ -129,10 +129,10 @@ const SearchModal = () => {
       />
       <CountrySelect
         value={location}
-        onChange={(value) => setLocation(value as CountrySelectValue)}
+        onCountryChange={(value) => setLocation(value)}
       />
       <hr />
-      <Map center={location?.latlng} />
+      <Map coordinates={[0, 0]} />
```

Comment:
Curious, why [0,0]

Reply:
You're right, I'm going to set the default location to Vancouver